octoprint_mrbeam/templates/tab_focus.jinja2

```diff
@@ -3,13 +3,13 @@
 						<div class="row-fluid">
 							<div class="span4">
 								<ul class="focus_steps">
-									<li>1. Place your material on the working area</li>
-									<li>2. Move the laser over the material</li>
-									<li>3. Put on your safety glasses</li>
-									<li>4. Turn the laser safety switch to on</li>
-									<li>&RightArrow; Now enable the focus mode</li>
-									<li>5. Adjust the focus until the laser beam is as small as possible</li>
-									<li>&RightArrow; Disable the focus mode. </li>
+									<li>{{ _('1. Place your material on the working area') }}</li>
```

Comment:
this file should not be translated as well. not used.

Reply:
ok, removed markings